common/libs/wallet/deterministic.ts

```diff
@@ -1,7 +1,7 @@
 import { IWallet } from './IWallet';
```

Comment:
Are we using this at all? And if we're not, should we?

Reply:
We're not using it in this file, so I'll remove the import. I don't think it's necessary here because it gets used further down in the inheritance chain (e.g. `LedgerWallet extends DeterministicWallet implements IWallet`).